lib/plugins/create/create.js

```diff
@@ -35,12 +35,13 @@ class Create {
 
     const validTemplates = [
       'aws-nodejs',
+      'aws-python',
     ];
 
-    if (validTemplates.indexOf(this.options.template)) {
+    if (validTemplates.indexOf(this.options.template) === -1) {
       const errorMessage = [
         `Template "${this.options.template}" is not supported.`,
-        ' Supported templates are: aws-nodejs.',
+        ' Supported templates are: "aws-nodejs" and "aws-python".',
```

Comment:
Imho would be better if we move `aws-nodejs and was-python` into a variable that can be used in both locations. Want to minimise the areas where we need to do changes